backend/penndata/views.py

```diff
@@ -416,7 +416,41 @@ def get(self, request, room_id):
                 "usage": {i: amt for i, amt in enumerate(usage_per_hour)},
             }
         )
+    
+class FitnessPreferences(APIView):
+    """
+    GET: returns list of a User's fitness preferences
+
+    POST: updates User fitness preferences by clearing past preferences
+    and resetting them with request data
+    """
+
+    permission_classes = [IsAuthenticated]
+
+    def get(self, request):
+
+        preferences = request.user.profile.fitness_preferences.all()
```

Comment:
hm is this guaranteed to exist?

Reply:
Yes! According to the `Profile` model in users. It will be empty upon the creation of a user profile